src/lib/components/workers-list.svelte

```diff
@@ -84,6 +83,7 @@
     </Table>
   {/if}
   <h2 class="text-base font-medium" data-testid="workers">
+    {workers?.pollers?.length || 0}
     {translate('workers', 'workers')}
   </h2>
```

Comment:
What do you think about using a `Badge` here instead?

| Before | After |
|--|--|
| <img width="381" alt="Screenshot 2023-08-30 at 12 19 00 PM" src="https://github.com/temporalio/ui/assets/15069288/ebb5354d-353f-4091-b72e-e3d7bdef3b29">|<img width="329" alt="Screenshot 2023-08-30 at 12 19 17 PM" src="https://github.com/temporalio/ui/assets/15069288/78ec749f-a157-4463-b753-f30c4268d02f">|

```suggestion
  <h2
    class="text-base font-medium flex items-center gap-2 mb-2"
    data-testid="workers"
  >
    {translate('workers', 'workers')}
    <Badge type="count" class="rounded-sm"
      >{workers?.pollers?.length || 0}</Badge
    >
  </h2>
```